dataprep/eda/missing/compute/nullivariate.py

```diff
@@ -14,42 +14,115 @@
 from ...intermediate import Intermediate
 from ...staged import staged
 
+__most__ = 5
```

Comment:
The parameter name is a little confusing. Could you come up with a more clear name? Besides, please try to avoid use global variables as parameters. I think these two parameters could be put into `plot_missing` function and then passed to other sub-functions. This part need to be handled by parameter management in the future.